.env.example

```
@@ -42,3 +42,6 @@ ORD_SERVER_URL="http://ipv4:port"
 
 GITHUB_CLIENT_SECRET=be8599360ca3e1234
 GITHUB_CLIENT_ID=Ov23liITA1234
+
+#Gemini API Key
+GEMINI_API_KEY=your_gemini_api_key_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)